packages/frontend/src/redux/crossStateSelectors/selectCollectedAvailableForClaimData.js

```diff
@@ -0,0 +1,81 @@
+import BN from 'bn.js';
+import { isEmpty } from 'lodash';
+import { createSelector } from 'reselect';
+
+import { selectValidatorsFarmData } from '../slices/staking';
+import {
+    selectTokensFiatValueUSD,
+    selectTokenWhiteList,
+} from '../slices/tokenFiatValues';
+import { selectAllContractMetadata } from '../slices/tokens';
+
+const collectFarmingData = (...args) => {
+    try {
+        if (args.some((el) => isEmpty(el))) return [];
+
+        const [
+            validatorsFarmData,
```

Comment:
Seems like there is a problem with `validatorsFarmData`, if you check with your account, you would see that when you first enter the Staking page there is only one farm - when you enter the third validator and get back the second farm is there.